CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed an issue that caused `UnrealGDK/Setup.sh` to report `sed: can't read : No such file or directory` when run on macOS.
 - Static subobjects on bNetLoadOnClient actors are now removed on clients in a manner matching native unreal's behavior. This change affects subobjects removed by the server while the actor is not in the client's interest.
 - Fixed an issue where multicast rpcs could be overwritten and then dropped on authority flicker.
+- When data for both an actor and its subobjects are received in a given tick, rep notifies will now be called on updated properties after the data has been applied for both the actor and its subobjects.
```

Comment:
This will now need to move to the above unreleased section

Reply:
good catch, moved